pages/login.js

```diff
@@ -1,11 +1,20 @@
 import Head from 'next/head';
+import { useRouter } from 'next/router';
 import { useState } from 'react';
 import styles from '../styles/Home.module.css';
+import { Amplify } from 'aws-amplify';
+import { withAuthenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+import awsExports from '../src/aws-exports';
+Amplify.configure(awsExports);
```

Comment:
Good for testing, but just don't forget to remove lines before we merge:
```
import { Amplify } from 'aws-amplify';
import awsExports from '../src/aws-exports';
Amplify.configure(awsExports);
```
These will be set up in our webapps entry point `_app.js` so the rest of the pages will have access to them.

Also I don't think `import '@aws-amplify/ui-react/styles.css';` is used?